contract-examples/test/utils.ts

```diff
@@ -0,0 +1,123 @@
+import { ethers } from "hardhat"
+const assert = require("assert")
+
+/*
+ *
+ * The `test` function is a wrapper around Mocha's `it` function. It provides a normalized framework for running the
+ * majority of your test assertions inside of a smart-contract, using `DS-Test`.
+ * The API can be used as follows:
+ * ```js
```

Comment:
Not sure if this annotation really makes sense here since it won't be picked up and show syntax highlighting as this isn't MD